pmm-app/src/pmm-qan/panel/components/Overview/components/QanTable/Table.styles.ts

```diff
@@ -72,22 +72,19 @@ export const getStyles = stylesFactory((theme: GrafanaTheme) => {
         position: sticky;
         left: 0;
         z-index: 1;
-        // outline: 1px solid ${borderColor};
       }
 
       .tr .td:nth-child(2) {
         position: -webkit-sticky; /* for Safari */
         position: sticky;
         left: 40px;
         z-index: 1;
-        // outline: 1px solid ${borderColor};
       }
       .th:first-child {
-        position: -webkit-sticky; /* for Safari */
-        position: sticky;
+        display: flex !important;
+        justify-content: center !important;
```

Comment:
I don't fully understand this change. Also, would it be possible to improve the specificity of the selector instead of using `!important` here?